resources/js/features/authentication/pages/Login.page.tsx

```diff
@@ -0,0 +1,45 @@
+import { Navigate } from "@solidjs/router";
+import { createQuery } from "@tanstack/solid-query";
+import { MemoLoader } from "components/ui";
+import { Page } from "components/utils";
+import { User } from "data-access/memo-api";
+import { Component, Match, Switch } from "solid-js";
+import { LoginForm } from "../forms/login";
+
+const LoginPage: Component = () => {
+  const statusQuery = createQuery(() => ({
+    ...User.statusQuery,
+    meta: { quietError: true },
```

Comment:
Dobra, widzę że tu coś do tego dodajesz. Może w takim razie `User.createStatusQuery({meta:{quietError:true}})`? No jakoś mi to nie leży :)

Reply:
możemy się posprzeczać, która forma jest lepsza :D